File: academia-dotNet/orientacao-a-objeto/aula-09/ConsumindoRestAPI/models/Pessoa.cs
namespace ConsumindoRestAPI.models;

public class Pessoa
{
    public int Id { get; set; }
    public string? Nome { get; set; }

    public Pessoa(string nome)
    {
        this.Nome = nome;
    }

    public override string? ToString()
    {
        return $"Id: {Id,-5} | Nome: {Nome,-15}";
    }
}

File: alura/ano-2024/java/java-api-httpclient/src/models/Pessoa.java
package models;

import mapping.PessoaMapping;

public class Pessoa {

    private String nome;
    private int idade;
    private String cidade;
    private String estado;

    public Pessoa(PessoaMapping pessoaMapping) {
        this.nome = pessoaMapping.nome();
        this.idade = pessoaMapping.idade();
        this.cidade = pessoaMapping.cidade();
        this.estado = pessoaMapping.uf();
    }

    @Override
    public String toString() {
        return "Pessoa [nome: " + nome + ", idade: " + idade + ", cidade: " + cidade + ", estado: " + estado + "]";
    }
}
